wrappers/javascript/nodejs/README.md

```diff
@@ -39,5 +30,7 @@ const getSchemaRequest = new GetSchemaRequest({
   schemaId: 'J6nTnUo3YLayzc2GUUctb1:2:MyName:1.0',
 })
 
-await pool.submitRequest({ requestHandle: getSchemaRequest.handle })
+const schemaResponse = await pool.submitRequest(getSchemaRequest)
```

Comment:
We don't get type inference here, right? I think we should either add the solution we discussed previously or we should add the generic to `pool.sumbitRequest<T>` in the example.